src/appengine/libs/issue_management/issue_filer.py

```diff
@@ -389,6 +391,11 @@ def file_issue(testcase,
                                       security_severity):
       issue.labels.add(result)
 
+  # Add additional body text from metadata.
+  additional_fields = json.loads(testcase.get_metadata('additional_fields', ''))
+  for key, value in six.iteritems(additional_fields):
+    issue.body += '\n\n%s: %s' % (key, value)
```

Comment:
nit: use python 3 f-strings.

e.g. `f'\n\n{key}: {value}`

Also, let's minimize new newlines introduced by appending the double newline once. and then doing a join on the rest on a single newline so that the final result looks lke:

```
...prev part of report

Field1:
Field2: 
```